### PR TITLE
Various manual updates

### PR DESCRIPTION
**Pull request by ekohl**
I started with going though the manual just to update the TFTP default but while reading through I found many more issues. Various of these should also be copied to older manuals but I first want to get some review so I don't need to go back and forth a few times.

### CODE REVIEW

File: _includes/manuals/1.24/3.3.1_rpm_packages.md [hunk 1]
```diff
@@ -48,9 +47,10 @@ Under each repo are directories for each distribution, then each architecture.
 
 To set up the repository, foreman-release packages are provided which add a repo definition to `/etc/yum.repos.d`.  Install the appropriate release RPM from these lists:
 
+    # EL7
+    yum localinstall https://yum.theforeman.org/releases/{{page.version}}/el7/x86_64/foreman-release.rpm
+    # EL8
     yum localinstall https://yum.theforeman.org/releases/{{page.version}}/el7/x86_64/foreman-release.rpm
```
Comment (tbrisker) [inline _includes/manuals/1.24/3.3.1_rpm_packages.md:51]:
```suggestion
    yum localinstall https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
```

**Review by tbrisker — COMMENTED** [review]
Just one tiny comment, otherwise looks good! thanks for the much-needed cleanup @ekohl !

File: _includes/manuals/1.24/3.3.1_rpm_packages.md (hunk 1 above)
Comment (ekohl) [reply _includes/manuals/1.24/3.3.1_rpm_packages.md:51]: So close ;)

**Review by tbrisker — APPROVED** [review]
Thanks @ekohl ! I'll let @ehelms add any comments, and then we'll need to make some of these in the current manuals as well.

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ Foreman API resource listings support pagination. Foreman 2.2 introduces 'all' a
 ### Upgrade warnings
 
 * Trends and statistics are now moved to separate plugin. See [manual](/plugins/foreman_statistics) of the new Statistics plugin if you wish continue using it.
+* TFTP integration is now [disabled by default](https://community.theforeman.org/t/disable-foreman-proxy-tftp-support-by-default/18892). Upgrades are unaffected. To enable TFTP integration, `--foreman-proxy-tftp true` must be passed to the installer command.
```
Comment (lzap) [inline _includes/manuals/nightly/1.2_release_notes.md:18]: Just for the record, this hasn't been merged yet. https://github.com/theforeman/puppet-foreman_proxy/pull/602

**Review by lzap — APPROVED** [review]

**Comment by ekohl** [comment]
Split off to https://github.com/theforeman/theforeman.org/pull/1628. Keeping this PR just for the TFTP changes.

**Review by ehelms — APPROVED** [review]

**Review by ehelms — APPROVED** [review]

**Comment by ekohl** [comment]
Ok, this can be merged since the TFTP change was merged.

**Pull request merged**